src/subscription_manager/gui/registergui.py

```diff
@@ -1007,7 +1090,28 @@ def pre(self):
                  (self.info.get_property('owner-key'),
                   self.info.get_property('environment'))
         self.info.set_property('register-status', msg)
+        self.async.register_consumer(self.info.get_property('consumername'),
+                                     self.facts,
+                                     self.info.get_property('owner-key'),
+                                     self.info.get_property('environment'),
+                                     self.info.get_property('activation-keys'),
+                                     self._on_registration_finished_cb)
+
+        return True
 
+
+class PerformForceRegisterScreen(PerformRegisterScreen):
+
+    def __init__(self, reg_info, async_backend, facts, parent_window):
+        super(PerformForceRegisterScreen, self).__init__(reg_info, async_backend, facts, parent_window)
+
+    def _on_unregistration_finished_cb(self, retval, error=None):
+        self.__register_consumer()
+
+    def _on_registration_finished_cb(self, retval, error=None):
```

Comment:
Not sure I understand this. Was this doing more before? I think it could be removed since it looks equivalent to
just inheriting the method. (Or is there some wacky class closure weirdness involved?)